metricbeat/module/kubernetes/apiserver/apiserver_test.go

```diff
@@ -31,8 +31,8 @@ func TestEventMapping(t *testing.T) {
 	ptest.TestMetricSet(t, "kubernetes", "apiserver",
 		ptest.TestCases{
 			{
-				MetricsFile:  "./_meta/test/metrics",
-				ExpectedFile: "./_meta/test/metrics.expected",
```

Comment:
it should be possible to keep old files isn't it? I would expect this to work both for old versions and the newer ones. What's the reasoning to remove these? Is it a no longer supported release maybe?

Reply:
good point, yes, I should get those back.
thanks!